backend/router/seriesRoute.js

```diff
@@ -0,0 +1,34 @@
+const express = require("express");
+const {
+  httpCreateSeries,
+  httpGetSeries,
+  httpGetSeriesById,
+  httpDeleteSeries,
+  httpUpdateSeries,
+  seriesLikeAndDislike,
+} = require("../controller/seriesController");
+
+// authentication and authorizations route
+const jwtAuth = require("../middleware/jwtAuth");
+const authAdmin = require("../middleware/authAdmin");
+const checkUserSubscription = require("../middleware/ckeckUserSubscribtion");
+
+const seriesRoute = express.Router();
+
+// create series route
+seriesRoute
+  .route("/")
+  .post(jwtAuth, authAdmin, httpCreateSeries)
+  .get(jwtAuth, checkUserSubscription, httpGetSeries);
+
+seriesRoute
+  .route("/:seriesId")
+  .get(jwtAuth, checkUserSubscription, httpGetSeriesById)
+  .delete(jwtAuth, authAdmin, httpDeleteSeries)
```

Comment:
I believe this authAdmin middleware has been changed, please check and update it

Reply:
seriesRoute is deleted.